Issue-Label-Bot-v2/notebooks/mlp.py

```diff
@@ -0,0 +1,171 @@
+from sklearn.neural_network import MLPClassifier
+from sklearn.model_selection import GridSearchCV
+from sklearn.metrics import roc_auc_score
+import dill as dpickle
+import numpy as np
+import pandas as pd
+
+
+class MLP:
```

Comment:
Can you add some test cases to test this class and also demonstrate how to use it?

Reply:
Okay!